src/box/relay_stream.c

```diff
@@ -0,0 +1,50 @@
+#include "relay_stream.h"
+#include "tweaks.h"
+#include "coio.h"
+#include "errinj.h"
+
+size_t relay_stream_flush_size = 16384;
+TWEAK_UINT(relay_stream_flush_size);
+
+void
+relay_stream_write(struct relay_stream *stream, const struct xrow_header *row)
+{
+	size_t used = region_used(&fiber()->gc);
+	int iovcnt;
+	struct iovec iov[XROW_IOVMAX];
+	xrow_to_iovec(row, iov, &iovcnt);
```

Comment:
Would it be possible to copy xrow directly to `lsregion`, bypassing `fiber->gc`?

Reply:
Yes, it's possible. Thanks for the suggestion!

The only problem I encountered is that I have to know the exact header size before I encode it (otherwise there'll be some garbage padding between consequent lsregion allocations, which will spoil the whole idea of dumping a whole slab at once).

So I added the `xrow_header_sizeof()` method. Unfortunately it duplicates the `xrow_header_encode()` almost completely, but I don't think there is anything we can do about it. I also thought about using macros here, like:
```c
#define ACTION(type, val) data = mp_encode_##type(data, val);
/* OR*/
#define ACTION(type, val) total_size += mp_sizeof_##type(val);

/* and replace all mp_encode_type(data, val) with ACTION(type, val) inside xrow_header_encode() */
```
But it seems this would make `xrow_header_encode` even uglier, so for now I went with the duplication approach.